scripts/mutect2_wdl/mutect2.wdl

```diff
@@ -175,10 +190,16 @@ workflow Mutect2 {
         File filtered_vcf = Filter.filtered_vcf
         File filtered_vcf_index = Filter.filtered_vcf_index
         File contamination_table = Filter.contamination_table
+        Array[String] tumor_bam_sample_names = M2.tumor_bam_sample_name
+        String tumor_bam_sample_name = tumor_bam_sample_names[0]
+        Array[String] normal_bam_sample_names = M2.normal_bam_sample_name
+        String normal_bam_sample_name = normal_bam_sample_names[0]
 
         # select_first() fails if nothing resolves to non-null, so putting in "null" for now.
         File? oncotated_m2_maf = select_first([oncotate_m2.oncotated_m2_maf, "null"])
         File? preadapter_detail_metrics = select_first([CollectSequencingArtifactMetrics.pre_adapter_metrics, "null"])
+        File? merged_bam_out = select_first([MergeBamOuts.merged_bam_out, "null"])
```

Comment:
I would call these `bamout` because the scatter and merge is just implementation.

Reply:
Done